gateway/modules/eventing/operations_test.go

```diff
@@ -0,0 +1,34 @@
+package eventing
+
+import (
+	"testing"
+
+	"github.com/spaceuptech/space-cloud/gateway/config"
+)
+
+func TestModule_SetRealtimeTriggers(t *testing.T) {
```

Comment:
Include the following test cases
- add a few eventing rules when no internal rules exist
- add a few eventing rules when a few non realtime eventing rules already exist
- add a few eventing rules when a few realtime eventing rule already exist
- add a few eventing rules when both realtime and non realtime rules already exist

In each test case, check if the final internal rules is set as expected

Reply:
How do i check the final result? My function doesn't return anything that i can assign as "got" to compare with want

Reply:
That's ok. You can check the struct directly